### PR TITLE
Build connection string to reform scan storage in code

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Build connection string to reform scan storage in code, so that it doesn't need to be manually set in Key Vault.

Rafał's existing PR in cnp-flux-config (https://github.com/hmcts/cnp-flux-config/pull/2201) already matches chart version from this PR and no other changes are needed.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/reform-scan-blob-router/values.yaml [hunk 1]
```diff
@@ -32,6 +31,7 @@ java:
     HANDLE_REJECTED_FILES_CRON: "0/30 * * * * *"
     TASK_SCAN_DELAY: "4000" # in millis
 
+    STORAGE_URL: https://reformscan.{{ .Values.global.environment }}.platform.hmcts.net
```
Comment (doncem) [inline charts/reform-scan-blob-router/values.yaml]: just double checking, is prod env included in this template?

Comment (lgonczar) [reply charts/reform-scan-blob-router/values.yaml]: Prod already overrides this setting: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/common/reform-scan/blob-router.yaml#L27

Comment (doncem) [reply charts/reform-scan-blob-router/values.yaml]: cool beans 👍

Comment (lgonczar) [reply charts/reform-scan-blob-router/values.yaml]: Deleted my already answered my comment by accident. Bringing it back:
> Prod already overrides this setting: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/common/reform-scan/blob-router.yaml#L27

File: src/main/java/uk/gov/hmcts/reform/blobrouter/config/StorageConfiguration.java [hunk 2]
```diff
@@ -31,9 +31,17 @@ public LeaseClientProvider getLeaseClientProvider() {
...
+        @Value("${storage.account-key}") String accountKey,
+        @Value("${storage.url") String storageUrl
     ) {
+        String connectionString = String.format(
+            "DefaultEndpointsProtocol=https;BlobEndpoint=%s;AccountName=%s;AccountKey=%s",
+            storageUrl,
+            accountName,
+            accountKey
+        );
+
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/config/StorageConfiguration.java]: I think we do not need to create connection string ,we can create the client by directly setting values.

Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/config/StorageConfiguration.java]: I think that's what Nitin tried initially and it didn't work. Is this correct @nitinprabhu ?

Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/config/StorageConfiguration.java]: > I think we do not need to create connection string ,we can create the client by directly setting values.

what do you mean @mdayican ? example ?

Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/config/StorageConfiguration.java]: I'm gonna merge this now, as this unblock production deployment. We can change the way we build the client at any point.

**Pull request merged**